Automattic/wp-calypso
# Updated stats notification style and ES6 syntax

Updated style of the Stats Notification in the 'Most popular day and hour' section (Insights). The position of the notification may not be ideal and require section redesign. Also updated the 'Chart' component to meet Linter requirements. 

**TO TEST:**

- Create a new WordPress.com site
- Navigate to **Stats / Insights** section and scroll down to 'Most popular day and hour' section
- You will see the style of the notification doesn't match with notifications in other sections:

![](https://cloudup.com/cXBAzWaY8Ly+)
Image Link: https://cloudup.com/cXBAzWaY8Ly

- To see the default styling of the notifications in other sections, navigate to **Stats / Days**:

![](https://cloudup.com/cC03o0v5XEK+)
Image Link: https://cloudup.com/cC03o0v5XEK

- After the PR, the notification in the 'Most popular day and hour' should look as follows (matching the rest of the sections):

![](https://cloudup.com/cjUd66k50qx+)
Image Link: https://cloudup.com/cjUd66k50qx

Tried to tweak the position of the notification (e.g. make it higher and lower). The final one (in PR) looks the best in my opinion but still not perfect. May need to redesign and hide the text on the background (covered by the notification but sticking out a bit at the top and bottom):

![](https://cloudup.com/cgd71rXjqdL+)
Image Link: https://cloudup.com/cgd71rXjqdL
